velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -891,8 +909,14 @@ core::TypedExprPtr ExpressionFuzzer::generateExpressionFromSignatureTemplate(
   ArgumentTypeFuzzer fuzzer{*chosen->signature, returnType, rng_};
   VELOX_CHECK_EQ(fuzzer.fuzzArgumentTypes(FLAGS_max_num_varargs), true);
   auto& argumentTypes = fuzzer.argumentTypes();
+  auto& constantArgs = fuzzer.constantArguments();
```

Comment:
Do we need to change ArgumentTypeFuzzer to pass through the constant flags? How about we keep ArgumentTypeFuzzer as is and get constant flags directly from chosen->signature?